Sources/FioriSwiftUICore/FioriButton/FioriButton.swift

```diff
@@ -53,6 +53,8 @@ import SwiftUI
 ///                 .background(Circle().fill(color))
 ///         }
 ///     }
+///
+///  To apply these styles to a `Button`, use `PrimaryButtonStyle`, `SecondaryButtonStyle` and `TertiaryButtonStyle` instead.
```

Comment:
```suggestion
///  To apply these styles to a `Button`, use `PrimaryButtonStyle`, `SecondaryButtonStyle`, and `TertiaryButtonStyle` instead.
```
I think you can safely delete the word "instead." It's not really necessary.